integration_tests/fixtures/forseti/modules/test_resources/main.tf

```diff
@@ -0,0 +1,64 @@
+/**
+* Copyright 2019 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+resource "random_id" "random_test_id" {
+  byte_length = 8
+}
+
+resource "random_pet" "random_name_generator" {
+}
+
+#-------------------------#
+# inventory-create.rb: Create KMS resources for inventory testing
+#-------------------------#
+resource "google_kms_key_ring" "test-keyring" {
+  project  = var.project_id
+  name     = "keyring-${random_pet.random_name_generator.id}"
+  location = "global"
+}
+
+resource "google_kms_crypto_key" "test-crypto-key" {
+  name            = "crypto-key-${random_pet.random_name_generator.id}"
+  key_ring        = google_kms_key_ring.test-keyring.self_link
+  rotation_period = "100000s"
+}
+
+#-------------------------#
+# scanner-bucket_acl_scanner.rb: Create a bucket with AllAuth + All Users Reader ACL
```

Comment:
Test resources are created within a separate module to keep things a little cleaner. Test resources should be associated with a specific test.